terminate called after throwing an instance of 'std::regex_error'
  what():  regex_error
bash: line 7:  9670 Aborted                 (core dumped) ./a.out
